lib/src/rules/use_setters_to_change_a_property.dart

```diff
@@ -62,7 +62,7 @@ class _Visitor extends SimpleAstVisitor {
         node.returnType?.type?.name != 'void') {
       return;
     }
-    bool _visitExpression(Expression expression) {
+    _visitExpression(Expression expression) {
```

Comment:
If the expectation is that it doesn't return any value, then we should use `void` rather than remove the return type.